src/main/java/org/example/examplefinalProject/controller/ClassRoomController.java

```diff
@@ -41,4 +46,12 @@ public ModelAndView detail(@PathVariable int classRoomId) {
         modelAndView.addObject("studentCount", classRoomService.studentCountById(classRoomId));
         return modelAndView;
     }
+
+    @PostMapping("/createClassRoom")
+    public String createClassRoom(@RequestParam String classRoomName,
+                                  @RequestParam String teacherFullName) {
+        classRoomService.createClassRoom(classRoomName, teacherFullName);
+        //log.info(String.format("User created teacher '%s' '%s'", teacherName, teacherSurname));
```

Comment:
Do we want to use this log or not? If not we could delete that.

Reply:
ano toto zatím používáme